lib/temporal/workflow/future.rb

```diff
@@ -3,7 +3,7 @@
 module Temporal
   class Workflow
     class Future
-      attr_reader :target, :callbacks
+      attr_reader :target, :callbacks, :exception
```

Comment:
Where is this being used?  My grep didn't turn anything up.

Reply:
There's existing code in the `execute!` method on `LocalWorkflowContext` (just below what I changed) that expected this to be public. It just wasn't being exercised by existing tests because failures bypassed the future before my change.

Reply:
It's convenient to expose this, but it can also be worked around if other folks are opposed to increasing the API surface area here, by doing the following in `.execute!` instead:
```
if future.failed?
  future.get
end
```